test/function/relative-external-include-once-nested/_config.js

```diff
@@ -0,0 +1,18 @@
+var assert = require( 'assert' );
+var path = require( 'path' );
+
+module.exports = {
+	description: 'includes a relative external module only once (nested version)',
+	options: {
+		external: path.join( __dirname, './first/foo.js' )
+	},
+	context: {
+		require: function ( required ) {
+			assert.equal( required, './first/foo.js' );
+			return 1;
+		}
+	},
+	exports: function ( exports ) {
+		assert.equal( exports, 3 );
+	}
+};
```

Comment:
I noticed that some of the test files don't have the trailing new line (for example the `consistent-renaming` tests) and even went through some pains to disable auto line adding in my editor because of that :smile: Will fix this in a jiffy.